assets/css/style.css

```diff
@@ -0,0 +1,225 @@
+/* base styles */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: inherit;
+}
+
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
+    font-weight: 300;
+}
+
+body{
+    font-size: 1.25rem;
+    font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+
+
+/* nav */
+
+
+.nav {
+    box-shadow: 0 2px 5px rgba(0, 0, 0, .1);
+    height: 4.3rem;
+    display: flex;
+    width: 73%;
+    background-color: #707070;
+    background: linear-gradient(rgb(131, 131, 131) 50%,rgb(112, 112, 112) 50%);
+    margin: 8.3rem auto;
+    border-radius: 4px;
+    border-bottom: 1px solid rgb(80, 80, 80);
+    border-top: 1px solid rgb(144, 144, 144);
+    box-shadow: rgba(0,0,0,0.3) 0 1px 2px;
+}
+
+.nav__list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+    width: 100%;
+    font-size: 1.4rem;
+}
+
+.nav__list li {
+    width: 100%;
+    padding: 1rem;
+    text-align: center;
+    border-left: 1px solid rgb(77, 76, 76);
+}
+
+.nav__list li:first-child {
+    border-left: none;
+    height: 100%;
+}
+
+.nav__list li:last-of-type {
+    border-left: none;
+}
+
+.nav-logo {
+    display: flex;
+    justify-content: center;
+    
+}
+
+.nav-logo img {
+    width: 2rem;
+}
+
+.nav__list li a {
+    color: rgb(233, 233, 233);
+    height: 100%;
+    text-decoration: none;
+}
+
+.nav__list li:not(:last-child):hover {
+    background-color: rgba(0, 0, 0, 0.38);
+}
+
+.nav-last {
+    border-right:1px solid rgb(77, 76, 76);
+}
+
+.search-bar {
+    display: flex;
+    justify-content: space-between;
+}
+
+input[type="text"]{
+    margin: 0 auto;
+    background: url(/assets/img/search_icon.png) 10px 50% no-repeat, linear-gradient(rgb(131, 131, 131) 70%,rgb(112, 112, 112) 50%) no-repeat;
+    background-size: 13px 13px;
+    border: none;
+    border-top: 1px solid rgb(80, 80, 80);
+    border-bottom: 1px solid rgb(144, 144, 144);
+    border-radius: 1.2rem;
+    height: 2.5rem;
+    width: 12rem;
+    padding-left: 2.7rem;
+    transition: width 0.4s;
+}
+
+input[type="text"]:focus{
+    outline: none;
+    width: 18rem;
+    background: url(/assets/img/search_icon2.png) 10px 50% no-repeat, white;
+    background-size: 13px 13px; 
+    transition: width 0.4s;
+}
+
+/* main content */
+
+.site__header {
+    padding: 1rem 0 3rem;
+  }
+  
+  .site__header.main {
+    background-image: url(/assets/img/your_verse_hero.jpg);
```

Comment:
Consider using `url(../img/your_verse_hero.jpg)` instead. check this comment for details.